LWOTC_WeaponPack/Localization/SMGs/XComGame.int

```diff
@@ -26,6 +26,6 @@ AbilityDescName="SMG"
 [Carbine_CV X2WeaponTemplate]
 FriendlyName="Carbine"
 FriendlyNamePlural="Carbines"
-BriefSummary="Carbine!!! Submachine guns offer a lighter, more concealable alternative to the standard assault rifle, although they lack the same punch and are less accurate at long range."
-TacticalText="<Bullet/>Carbine!!! Firing the primary weapon ends the soldier's turn, in the absence of special training.\n<Bullet/> Use upgrades recovered from ADVENT enemies to improve the SMG's tactical effectiveness. \n<Bullet/> SMG-style weapons offer bonus mobility, close-quarters accuracy and concealment relative to assault rifles, but they do less damage, have less ammo, and suffer accuracy penalties at long range."
-AbilityDescName="SMG"
+BriefSummary="Carbine offer a lighter and more accurat alternative to the standard smg, although they lack the same concealabilety."
```

Comment:
Carbines offer a heavier and more accurate alternative to the standard SMG, although they lack the same concealability.